common/src/test/java/com/revenuecat/purchases/common/BackendTest.kt

```diff
@@ -1377,8 +1378,6 @@ class BackendTest {
             "price" to receiptInfo.price,
             "currency" to receiptInfo.currency,
             "normal_duration" to receiptInfo.duration,
-            "intro_duration" to receiptInfo.introDuration,
-            "trial_duration" to receiptInfo.trialDuration,
```

Comment:
Removing these for now. Still need to check if the response is sending the pricing phases. If so, I will add them here.